maxim/logger/together/helpers.py

```
@@ -101,6 +101,7 @@ def sync_stream_helper(
                 trace.end()
 
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)